src/EnergyPlus/DataSizing.cc

```diff
@@ -269,7 +269,7 @@ namespace DataSizing {
     Real64 DataCoilSizingAirOutHumRat(0.0);         // saves sizing data for use in coil object reporting
     Real64 DataCoilSizingFanCoolLoad(0.0);          // saves sizing data for use in coil object reporting
     Real64 DataCoilSizingCapFT(1.0);                // saves sizing data for use in coil object reporting
-    Real64 DataDesAccountForFanHeat(true);          // include fan heat when true
+    bool DataDesAccountForFanHeat(true);            // include fan heat when true
```

Comment:
Well this is certainly an improvement already... :upside_down_face:

Reply:
Fixed my original mistake. Thanks to the compiler is was working then as well.